### DISCUSSION TITLE
[go-cqhttp/ERROR] PyCqBot: go-cqhttp 发生致命错误 登录时发生致命错误: Packet failed to sendPacket: connection closed如何解决

### DISCUSSION THREAD

**Discussion by v58758** [Moemu/Muice-Chatbot#14] [Q&A]
RT，求解决方案

**Comment by v58758** [comment]
验证完后报错

**Reply by Moemu** [reply]
非本项目源码问题，请移至[go-cqhttp](https://github.com/Mrs4s/go-cqhttp/issues)提问

**Reply by Moemu** [reply]
10.20更新：我们已无力提供对qqbot相关代码的更新，详见https://github.com/Moemu/Muice-Chatbot/issues/18 ，目前我们打算提供一个前端页面来完成对沐雪的任何交互，对此带来的不便我深感歉意。
